apps/explorer/lib/explorer/chain/import/runner/blocks.ex

```diff
@@ -385,219 +347,6 @@ defmodule Explorer.Chain.Import.Runner.Blocks do
     end
   end
 
-  defp delete_address_token_balances(repo, consensus_block_numbers, %{timeout: timeout}) do
```

Comment:
these are all unused?

Reply:
Yeah, those were referenced from the commented out code.